chart/README.MD

```diff
@@ -0,0 +1,24 @@
+# ToCloud9 Helm chart
+
+This chart will deploy a ToCloud9 cluster using **AzerothCore** as backend.
+
+> **WARNING:_** 
+Pre-built images in this chart values won't be actively maintained, build your own ones if you want to be up to date with AzerothCore releases.
```

Comment:
It would be nice to configure a pipeline in this repository that builds images and pushes them into this registry - https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry.